components/script/dom/cssstylesheet.rs

```diff
@@ -36,6 +38,7 @@ impl CSSStyleSheet {
             owner: JS::from_ref(owner),
             rulelist: MutNullableJS::new(None),
             style_stylesheet: stylesheet,
+            origin_clean: Cell::new(false),
```

Comment:
We should probably assume this is clean instead, or we'll break inline stylesheets and the ones from meta elements unless we add special handling for them.